addons/l10n_ch/views/account_invoice_view.xml

```diff
@@ -1,45 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <record id="isr_invoice_form" model="ir.ui.view">
+        <record id="qrr_invoice_form" model="ir.ui.view">
             <field name="name">l10n_ch.account.invoice.form</field>
             <field name="model">account.move</field>
             <field name="inherit_id" ref="account.view_move_form"/>
             <field name="arch" type="xml">
                 <xpath expr="//field[@name='is_move_sent']" position="after">
-                    <field name="l10n_ch_isr_sent" invisible="1"/>
+                    <field name="l10n_ch_qrr_sent" invisible="1"/>
                     <field name="l10n_ch_currency_name" invisible="1" readonly="1"/>
                     <field name="l10n_ch_is_qr_valid" invisible="1"/>
                 </xpath>
                 <header position="after">
-                    <field name="l10n_ch_isr_needs_fixing" invisible="1"/>
-                    <div groups="account.group_account_invoice" class="alert alert-warning" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('l10n_ch_isr_needs_fixing', '=', False)]}">
-                        Please fill in a correct ISR reference in the payment reference.  The banks will refuse your payment file otherwise.
+                    <field name="l10n_ch_qrr_needs_fixing" invisible="1"/>
+                    <div groups="account.group_account_invoice" class="alert alert-warning" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('l10n_ch_qrr_needs_fixing', '=', False)]}">
+                        Please fill in a correct QRR reference in the payment reference.  The banks will refuse your payment file otherwise.
```

Comment:
"The banks will refuse your payment file otherwise" ? I know it was there before, but what file are we talking about ? What payment ? It seems weird to me to be talking about a payment file somehow made by the company ... on a customer invoice. Should we rephrase it ? Or just drop it ? Check with a PO !

Reply:
I'll ask, but since other blocking pop-up have been added when talking about the QR generation, I'm not even sure that this can be displayed anymore